src/core/client/stream/tabs/Live/LiveConversation/LiveConversationQuery.tsx

```diff
@@ -36,9 +43,16 @@ const LiveConversationQuery: FunctionComponent<Props> = ({
   settings,
   comment,
   onClose,
+  highlightedCommentState,
 }) => {
   const { relayEnvironment } = useCoralContext();
-  const [cursor, setCursor] = useState(new Date(0).toISOString());
+  const [cursor, setCursor] = useState(
+    highlightedCommentState
+      ? new Date(
+          new Date(highlightedCommentState.cursor).getTime() - 10
```

Comment:
Don't substract by `- 10` it's a hack. Instead fix what's wrong with the query. In the Convo View the Before Container should always set `inclusive: false` and the After Container to `inclusive: true`. At the moment this is flipped around which is wrong.